### PR TITLE
Adds address_id to log_value

### PR DESCRIPTION
**Pull request by yaoandrew**

### CODE REVIEW

**Review by rmulhol — CHANGES_REQUESTED** [review]
Adding this temporarily since it's a breaking change

**Comment by paytonrules** [comment]
@yaoandrew - mostly so it doesn't get lost, I bet you could rebase this to breaking-changes.

File: transformers/events/log_value/transformer.go [hunk 1]
```diff
@@ -17,67 +17,48 @@
...
 	"github.com/makerdao/vdb-mcd-transformers/transformers/shared"
 	"github.com/makerdao/vulcanizedb/libraries/shared/factories/event"
 	"github.com/makerdao/vulcanizedb/pkg/core"
 	"github.com/makerdao/vulcanizedb/pkg/datastore/postgres"
-	"github.com/makerdao/vulcanizedb/pkg/eth"
 )
 
 type Transformer struct{}
 
 const Val event.ColumnName = "val"
```
Comment (rmulhol) [inline transformers/events/log_value/transformer.go]: should we delete this now?

File: transformers/events/log_value/transformer.go [hunk 2]
```diff
@@ -17,67 +17,48 @@
...
+		contractAddressID, contractAddressErr := shared.GetOrCreateAddress(log.Log.Address.String(), db)
+		if contractAddressErr != nil {
+			return nil, err
+		}
 
-	var models []event.InsertionModel
-	for _, logValueEntity := range entities {
-		bigIntVal := new(big.Int).SetBytes(logValueEntity.Val[:])
+		bigIntVal := new(big.Int)
+		bigIntVal.SetBytes(log.Log.Data)
```
Comment (rmulhol) [inline transformers/events/log_value/transformer.go:45]: mostly just curious - what inspired you to take out parsing of the entity from the ABI? This seems more elegant - I hadn't even realized we could do this 🎉

File: transformers/events/log_value/transformer.go [hunk 3]
```diff
@@ -17,67 +17,48 @@
...
-	"github.com/makerdao/vulcanizedb/pkg/eth"
 )
 
 type Transformer struct{}
 
 const Val event.ColumnName = "val"
 
-func (Transformer) toEntities(contractAbi string, logs []core.EventLog) ([]LogValueEntity, error) {
-	var entities []LogValueEntity
+func (t Transformer) ToModels(abi string, logs []core.EventLog, db *postgres.DB) ([]event.InsertionModel, error) {
```
Comment (rmulhol) [inline transformers/events/log_value/transformer.go]: maybe worth replacing `abi` with `_`, if it's no longer used

File: transformers/events/log_value/transformer.go [hunk 4]
```diff
@@ -17,67 +17,48 @@
 package log_value
 
 import (
-	"fmt"
 	"math/big"
 
 	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
```
Comment (rmulhol) [inline transformers/events/log_value/transformer.go]: super nbd but could prob remove this whitespace

File: db/migrations/20191118031513_create_log_value.sql [hunk 5]
```diff
@@ -4,6 +4,7 @@ CREATE TABLE maker.log_value
     id        SERIAL PRIMARY KEY,
     header_id INTEGER NOT NULL REFERENCES public.headers (id) ON DELETE CASCADE,
     log_id    BIGINT  NOT NULL REFERENCES public.event_logs (id) ON DELETE CASCADE,
+    address_id BIGINT NOT NULL REFERENCES public.addresses (id) ON DELETE CASCADE,
```
Comment (rmulhol) [inline db/migrations/20191118031513_create_log_value.sql]: could "Reformat Code" to fix the alignment

**Review by rmulhol — COMMENTED** [review]
:shipit:

**Review by rmulhol — APPROVED** [review]

File: transformers/events/log_value/transformer.go (hunk 2 above)
Comment (yaoandrew) [reply transformers/events/log_value/transformer.go:45]: 😄 I'm glad it works, I think I just noticed that during the refactor that if all I needed was the data, I could grab it from the log.

**Pull request merged**